layers/sqlsrv/test.php

```diff
@@ -17,7 +17,7 @@
 // This test attempts a connection, forcing MS odbc libraries to be loaded
 if (!sqlsrv_connect('localhost', ['LoginTimeout' => 1])) {
     foreach (sqlsrv_errors() as $error) {
-        if ($error['SQLSTATE'] == '01000') {
+        if ($error['SQLSTATE'] == '01000' || $error['SQLSTATE'] == 'IM002') {
```

Comment:
This makes the build fail, as mentioned in https://github.com/brefphp/extra-php-extensions/issues/418#issuecomment-1528873718. It’s a start.